shared/src/commonMain/kotlin/com/mbta/tid/mbta_app/model/SheetRoutes.kt

```diff
@@ -58,11 +58,15 @@ sealed class SheetRoutes {
             }
         }
 
-        /** When transitioning between certain routes, we don't want to resize the sheet */
+        /**
+         * We want to retain sheet size unless moving into or out of stop details, or between tabs
+         */
```

Comment:
Updates based on discussion [with design in figma](https://www.figma.com/design/OVO5Z0eYUtnnRvbxsIRpO3?node-id=2414-128076#1365777321)